hydra-node/src/Hydra/Node.hs

```diff
@@ -347,7 +347,6 @@ processEffects node tracer inputId effects = do
   processEffect (effect, effectId) = do
     traceWith tracer $ BeginEffect party inputId effectId effect
     case effect of
-      ClientEffect i -> sendOutput server i
```

Comment:
We should still keep the `ClientEffect` for the purpose of sending some messages to the client that we don't want persisted. Also keep the `GetUTxOResponse`.

Reply:
I ended up removing it since it requires `ServerOutput` and in turn we need to accept the same type in `Server{sendOutput}` and now we accept `StateChanged`. We could keep track of `seq` numbers and do it this way but our original plan was to actually remove `ClientEffect` so I ended up doing that.